harry-core/src/harry/runner/LockingDataTracker.java

```diff
@@ -65,7 +65,7 @@ public void beginModification(long lts)
     {
```

Comment:
Unrelated to the patch I guess, but should `readingFrom` and `writingTo` be final above?

Reply:
yes, final not static! 👍